### ISSUE TITLE
horrible use description.

### ISSUE THREAD

**Issue by lbilodeau** [Azure/azure-sdk-for-net#37681]
nothing but jargon, no examples of usage. no explaination on how to use it in conjunction to the DefaultAzureCredential class.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 8279aa91-58d0-7319-86fa-b67c07e3f3bd
* Version Independent ID: 3324e0cc-7a80-d42a-0ac5-1dc70fbea481
* Content: [DefaultAzureCredentialOptions.AdditionallyAllowedTenants Property (Azure.Identity) - Azure for .NET Developers](https://learn.microsoft.com/en-us/dotnet/api/azure.identity.defaultazurecredentialoptions.additionallyallowedtenants?view=azure-dotnet#azure-identity-defaultazurecredentialoptions-additionallyallowedtenants)
* Content Source: [xml/Azure.Identity/DefaultAzureCredentialOptions.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/master/xml/Azure.Identity/DefaultAzureCredentialOptions.xml)
* Service: **azure**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by jsquire** [comment]
Thank you for your feedback.  Tagging and routing to the team member best able to assist.

**Comment by christothes** [comment]
Hi @lbilodeau - Did you have any specific suggestion for how to improve this? Reading it again, it seems like straight forward language without any complicated terminology.

**Comment by christothes** [comment]
Hi @lbilodeau - Did you have any specific suggestion for how to improve this? Reading through it again, it seems like straight forward language without any complicated terminology.

**Comment by lbilodeau** [comment]
When reading that page it was not clear to me that the DefaultAzureCredentialOptions was a parameter argument for the DefaultAzureCredential class. When I read that page I got the idea that that was a method for that class.

Regardless I’m still not able to access my keyvault using the service principles settings I added to my environment variables from VS code. The error I get looks to be pulling the tenant ID from someplace other than the environment values. This is my latest attempt.

app_credentials = DefaultAzureCredential(logging_enable=True, DefaultAzureCredentialOptions = "*", exclude_cli_credential = True,
                                                         exclude_managed_identity_credential = True,
                                                         exclude_workload_identity_credential = True,
                                                         exclude_developer_cli_credential = True)

this is the error I get:
azure.core.exceptions.ClientAuthenticationError: The current credential is not configured to acquire tokens for tenant 10bfc8b9-364b-4013-b36f-cef0ace33dbe. To enable acquiring tokens for this tenant add it to the additionally_allowed_tenants when creating the credential, or add "*" to additionally_allowed_tenants to allow acquiring tokens for any tenant.
DefaultAzureCredential failed to retrieve a token from the included credentials.

where my environment setting for the tenant ID is: "AZURE_TENANT_ID" : "1b6e7e27-85b2-4e98-a0b0-f5fafb2bb005",

Am I missing a setting someplace?

-Larry Bilodeau
